code/modules/vore/resizing/resize_vr.dm

```diff
@@ -112,15 +112,15 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
  * @return false if normal code should continue, 1 to prevent normal code.
  */
 /mob/living/proc/attempt_to_scoop(var/mob/living/carbon/human/H)
-	var/size_diff = H.get_effective_size() - src.get_effective_size()
+	var/size_diff = src.get_effective_size() / H.get_effective_size()
```

Comment:
Looking at the math now.

src.get_effective_size() / H.get_effective_size()

SRC is the mob trying to pick up someone. H is the person being picked up.

The minimum size to pick someone up is 50% size difference.
NEW version: (If size diff is >= 2, pick them up)
2.0/1.5 = 1.333
1.5/1 = 1.5
1.0/0.5 = 2.0
0.75/0.25 = 3.0

OLD version: (If size diff is >= .5, pick them up)
2.0-1.5 = 0.5
1.5-1 = 0.5
1.0-0.5 = 0.5
0.75-0.25 = 0.5


If I'm reading this correctly, this would mean that large characters can't pick up people that are ~50% of them, making the micro pickup/crush code not work.

Reply:
Anyone can pick up mobs that are at most half the size of themself with this change, simply put.

Reply:
>Anyone can pick up mobs that are at most half the size of themself

I can't support this. 200% should be able to pick up 150% and 150% should be able to pick up 100%. If they can't do that, this PR has absolutely no support from me.

Reply:
>Anyone can pick up mobs that are at most half the size of themself with this change, simply put.

# This is a hidden and massive nerf that wasn't mentioned in the PR description. 

Not once did it mention that in the PR. The closest thing you said was "Now uses relative size difference (instead of absolute) for checks and damage dealing."

This isn't "fixes and tweaks" this is just a flat out nerf. This means that 200% can't pick up 150, and 150% can't pick up 100%

# 200% ALSO can't step on 101%

This huge nerf was hidden and not mentioned at all, which **is not fucking OK**

Reply:
I didn't hide the fact that the system was swapped to using a relative difference calculations. It implies that there will be changes.

What coefficient is used is completely open to debate and you can make suggestions to it.

No need for using title fonts, I can read.

Reply:
The issue with making it check for pred_heigh/prey_height is that it causes extremely odd interactions across the board, while having it check for 50% difference worked well  and provided enough of a size difference.

200/150 = 1.33. 200/150 = 1.5 Making it 1.3 works perfectly in these two cases. 
200/153 = 1.307. This means 200% can pick up _just_ a bit higher than before. Not that big of a change.

But as you start to get lower sizes, things start to get crazy.

0.100/0.76 = 1.31. 100% would be able to pick up 76%, and there's only a 24% size diff here.
0.75/0.57 = 1.31 75% would be able to pick up 57%, which is only a 18% size difference.
0.50/0.38 = 1.31 50% would be able to pick up 38%, which is again a 12% size difference
0.33/0.25 = 1.32 33% would be able to pick up 25%, which is only a 8% size difference.

Trying to get it to work with relative size is good, but it just... Really doesn't work out. If you increase it, 200% can't pick up 150%. Having it be a 50% size difference makes it so it's even across the board and you don't get crazy things like, say, a micro managing to step on another micro when they're almost the exact same size.

Also, I do apologize for the boldfont. I was under the impression that the size changes to make it so, say, 200% can't step on anyone above 100% were being snuck in intentionally without including info about it.

Reply:
I understand the sentiment. Though a set difference of 0.75 makes it so you can only step on 0.25 when normal-sized, and not on anything when being the slightest bit smaller.

If sizes past 200% were allowed the interactions among large characters would become quite extreme as well, but since those sizes will most likely remain inaccessable I guess the model is good enough for that.

So, since 200% being able to interact with 150% is apparently wanted, how about a coefficient of 1.3 for a victim of size 100% or greater, and 2.0 for victims below that size?

Reply:
Doing 1.3 for 100% or greater and 2.0 for victims below 100% could work, but then the issue of 99% requiring someone to be 198 and such arises, which would result in a whole slew of issues. Maybe make it so... 
>= 75% = 1.3 (Looking back at this, it could possibly be increased to 1.4. Make it so 200 can pick up <= 140, resulting in a small decrease in pickup size while not lowering it too much.) 
< 75 && >50 (This is where it's somewhat annoying to get the calculations.) =1.6
<= 50% = 2.0

Reply:
On second thought, at this point the mathematical model used is becoming unnecessary convoluted. It should be kept at a simple x % size advantage or x size difference.